src/Ombi/ClientApp/src/app/requests-list/components/requests-list.component.scss

```
@@ -1,72 +1,120 @@
-@media (min-width: 1170px){
-	.justify-content-md-center {
-    	justify-content: center !important;
-	}
+// Tab Content
+.tab-content {
+  animation: fadeIn 0.25s ease;
 }
 
-::ng-deep table.requests button{
-	margin:5px;
+@keyframes fadeIn {
```

Stylelint (17.6.0)

[error] Expected keyframe name "fadeIn" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)